README.md

```diff
@@ -135,46 +195,68 @@ metrics.describe(
     },
 );
 ```
- 
 
+### Clustered aggregation strategy
 
-### Gauge
-
-Used to record a value which can vary over time, like temperature.
-
-```
-metrics.gauge('greenhouse_temperature', temp [, labelObject ]);
-```
+There are several aggregation strategies which `prom-client`'s `AggregatorRegistry`
+can use to combine the metrics recorded by `cluster` workers. They are:
 
-#### Counter
-
-Used to record increases to a monotonic counter, like requests served.
+- sum
+- first
+- min
+- max
+- average
+- omit
```

Comment:
Are these the only strategies that can be selected? I wonder if then it should be added to the typings as an Enum for example, so people wouldn't set a wrong one by accident? (I'm just guessing)

Reply:
Great point @imrehg I'll see about this

Reply:
Looks good on this port, resolving this comment